docs/pbc/all/tax-management/202400.0/third-party-integrations/vertex/install-vertex.md

```diff
@@ -627,3 +627,53 @@ class ProductOfferStockDependencyProvider extends SprykerProductOfferStockDepend
 }
 
 ```
+
+
+---
+
+
+## Appendix A
+
+### Quote/Order object to Vertex API mapping
+
+Mapping of Spryker Quote/Order transfer object to Vertex API request format is described in the following table:
+
```

Comment:
@bohdan-turchyk-spryker can you please provide a few words with the context of why we have this mapping, e.g. when to use it?

Reply:
> Please follow this data mapping format defined here if you need to implement any custom expander plugins for Vertex integration.

I can add this line in the doc if needed.

Reply:
Added an explanation, please check.